include/cucumber-cpp/internal/step/BreakpointSupport.hpp

```diff
@@ -0,0 +1,40 @@
+#ifndef CUKE_BREAKPOINTSUPPORT_HPP_
+#define CUKE_BREAKPOINTSUPPORT_HPP_
+
+#include "../CukeDll.hpp"
+
+namespace cucumber {
+namespace internal {
+
+#if defined(_MSC_VER)
+// The ", true" makes the expression to be of type bool, which is required
+// for the CUKE_BREAK_INTO_DEBUGGER macro
+#define CUKE_BREAK_INTO_DEBUGGER (__debugbreak(), true)
```

Comment:
No, we aren't. I'll remove the comment and change the macro to just be `__debugbreak()`. Should I also change the mac part to be just `{ asm("int3"); }`?